draft/2022-05-11-this-week-in-rust.md

```diff
@@ -27,34 +27,60 @@ and just ask the editors to select the category.
 
 -->
 
+
 ### Official
 
+* [Security advisory: malicious crate rustdecimal](https://blog.rust-lang.org/2022/05/10/malicious-crate-rustdecimal.html)
+* [CTCFT 2022-05-16 Agenda](https://blog.rust-lang.org/inside-rust/2022/05/10/CTCFT-may.html)
+
 ### Foundation
 
 ### Newsletters
 
 ### Project/Tooling Updates
 
+* [rust-analyzer changelog #128](https://rust-analyzer.github.io/thisweek/2022/05/09/changelog-128.html)
 * [Fornjot 0.6](https://www.fornjot.app/blog/fornjot-0.6/)
 * [Announcing the Kani Rust Verifier Project](https://model-checking.github.io/kani-verifier-blog/2022/05/04/announcing-the-kani-rust-verifier-project.html)
 * [Slint (UI crate) weekly update -- Version 0.2.2 Release](https://slint-ui.com/thisweek/2022-05-09.html)
+* [Rocket's 2nd v0.5 Release Candidate - Rocket Web Framework](https://rocket.rs/v0.5-rc/news/2022-05-09-version-0.5-rc.2/)
+* [rustc_codegen_gcc: Progress Report #11](https://blog.antoyo.xyz/rustc_codegen_gcc-progress-report-11)
+* [GCC Rust Monthly Report #16 April 2022](https://thephilbert.io/2022/05/04/gcc-rust-monthly-report-16-april-2022/)
+* [RepliByte - An open-source tool to seed your dev database with real data](https://www.reddit.com/r/rust/comments/ukmnow/an_opensource_tool_to_seed_your_dev_database_with/)
+* [This Month in Rust OSDev: April 2022](https://rust-osdev.com/this-month/2022-04/)
```

Comment:
Maybe this should go in Newsletters?

Reply:
I'll move this to Newsletters this time, and if you disagree we can discuss on Discord for next time :)

Reply:
I do think that should be in Newsletters as well